Common subdirectories: kdelibs.orig/admin and kdelibs/admin
Common subdirectories: kdelibs.orig/arts and kdelibs/arts
Common subdirectories: kdelibs.orig/dcop and kdelibs/dcop
Common subdirectories: kdelibs.orig/debian and kdelibs/debian
Common subdirectories: kdelibs.orig/doc and kdelibs/doc
Common subdirectories: kdelibs.orig/interfaces and kdelibs/interfaces
Common subdirectories: kdelibs.orig/kab and kdelibs/kab
Common subdirectories: kdelibs.orig/kabc and kdelibs/kabc
Common subdirectories: kdelibs.orig/kate and kdelibs/kate
Common subdirectories: kdelibs.orig/kcert and kdelibs/kcert
Common subdirectories: kdelibs.orig/kconf_update and kdelibs/kconf_update
diff -u -x CVS kdelibs.orig/kde.pot kdelibs/kde.pot
--- kdelibs.orig/kde.pot	Sun May 30 10:36:17 2004
+++ kdelibs/kde.pot	Wed Jun  9 09:44:11 2004
@@ -6,7 +6,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"POT-Creation-Date: 2004-05-30 10:36+0200\n"
+"POT-Creation-Date: 2004-06-09 09:43+0200\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
Common subdirectories: kdelibs.orig/kdecore and kdelibs/kdecore
Common subdirectories: kdelibs.orig/kded and kdelibs/kded
Common subdirectories: kdelibs.orig/kdefx and kdelibs/kdefx
Common subdirectories: kdelibs.orig/kdeprint and kdelibs/kdeprint
Common subdirectories: kdelibs.orig/kdesu and kdelibs/kdesu
Common subdirectories: kdelibs.orig/kdeui and kdelibs/kdeui
Common subdirectories: kdelibs.orig/kdewidgets and kdelibs/kdewidgets
Common subdirectories: kdelibs.orig/kdoctools and kdelibs/kdoctools
Common subdirectories: kdelibs.orig/khtml and kdelibs/khtml
Common subdirectories: kdelibs.orig/kimgio and kdelibs/kimgio
Common subdirectories: kdelibs.orig/kinit and kdelibs/kinit
Common subdirectories: kdelibs.orig/kio and kdelibs/kio
Common subdirectories: kdelibs.orig/kioslave and kdelibs/kioslave
Common subdirectories: kdelibs.orig/kjs and kdelibs/kjs
Common subdirectories: kdelibs.orig/kmdi and kdelibs/kmdi
Common subdirectories: kdelibs.orig/kparts and kdelibs/kparts
Common subdirectories: kdelibs.orig/kresources and kdelibs/kresources
Common subdirectories: kdelibs.orig/ksmartcard and kdelibs/ksmartcard
Common subdirectories: kdelibs.orig/kstyles and kdelibs/kstyles
Common subdirectories: kdelibs.orig/kutils and kdelibs/kutils
Common subdirectories: kdelibs.orig/kwallet and kdelibs/kwallet
Common subdirectories: kdelibs.orig/libkmid and kdelibs/libkmid
Common subdirectories: kdelibs.orig/libkscreensaver and kdelibs/libkscreensaver
Common subdirectories: kdelibs.orig/libltdl and kdelibs/libltdl
Common subdirectories: kdelibs.orig/licenses and kdelibs/licenses
Common subdirectories: kdelibs.orig/mimetypes and kdelibs/mimetypes
Common subdirectories: kdelibs.orig/pics and kdelibs/pics
